Packs/Jira/Integrations/JiraAuditRecords/JiraAuditRecords.yml

```diff
@@ -0,0 +1,113 @@
+category: Analytics & SIEM
+commonfields:
+  id: JiraAuditRecords
```

Comment:
Change integration id, name, and display to `Jira Events Collector`.
Also, Update the folder/file names.